crates/ln-dlc-node/src/channel.rs

```diff
@@ -37,10 +36,7 @@ pub struct Channel {
     /// the funding transaction. We use the `user_channel_id` as identifier over the entirety
     /// of the channel lifecycle.
     pub channel_id: Option<ChannelId>,
-    /// The fake short channel id (generated for intercept invoice)
-    ///
-    /// This is only `Some` for JIT channels.
-    pub fake_scid: Option<FakeScid>,
+    pub liquidity_option_id: i32,
```

Comment:
:question: Can this be filled for non-JIT channels?

Reply:
I guess not.. I'll change it to an option.

Reply:
but it will mean we can't define a foreign key. :/